storage/azure/src/main/java/io/aiven/kafka/tieredstorage/storage/azure/AzureBlobStorageConfig.java

```diff
@@ -92,7 +96,14 @@ public class AzureBlobStorageConfig extends AbstractConfig {
                 null,
                 Null.or(new NonEmptyPassword()),
                 ConfigDef.Importance.MEDIUM,
-                AZURE_CONNECTION_STRING_DOC);
+                AZURE_CONNECTION_STRING_DOC)
+            .define(
+                AZURE_UPLOAD_BLOCK_SIZE_CONFIG,
+                ConfigDef.Type.INT,
+                AZURE_UPLOAD_BLOCK_SIZE_DEFAULT,
+                ConfigDef.Range.atLeast(1),
```

Comment:
Azure doesn't allow more than 50K blocks for one apload. Having 1 bytes won't allow uploading anything useful. We need at least 42 KiB to upload the biggest segment (2 GiB). I think we should stick to min 100 KiB here.